script/Ciel/Day 9/09_01_CIEL09_6_1.txt

```diff
@@ -171,57 +187,64 @@ Arcueid nonchalantly moves closer.
 [sha:e943ef9a845440d6598fd28d7f2837caffb9c46e]{
 -- Page 680, Offset 23902.
 -- もっとも、あの時の再現はありえないけど。
-Mind you, there won't be any reenactment of what happened the last time.
+Not that you'd be able to pull it off a second time anyway, mind you.
 }
 [sha:d36b1f3887b994ad4943eb0b2a748471092a1ec2]{
 -- Page 680, Offset 23903.
 -- いくら貴方の直死の眼でも、今のわたしから死を読み取ることはできないでしょう？」
-In my current state, even your Eyes of Death Perception can't make out any traces of death on my body, am I right?"
+Even with your Eyes of Death Perception,
+ I'm willing to bet you can't perceive my death at all right now, can you?"
 }
 [sha:3380e3408dec09f88cddbcc60350ed2ba317c1a2]{
 -- Page 681, Offset 23904.
 -- 真正面から。
-She's facing me directly.
+We're face to face now.
 }
 [sha:53e147ecd7d004a04644973ea664a42f50044968]{
 -- Page 681, Offset 23905.
 -- 金の瞳が、俺の眼球に飛び込んできた。
-Her golden pupils plunge into my eyeballs.
+// Nasu learn anatomy challenge, part ■■■
+// This one feels pretty weird to TL literally - her eyes aren't really rushing into his so much as her _gaze_
+Her golden eyes transfix me with their stare.
```

Comment:
I think it needs to be vaguely creepy. That's why he went with this weird-ass description. This is when the Arc of her looking at him pops up.
>Her golden eyes dig deep into my own.